api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/context/MessageContextLoaderTest.java

```diff
@@ -144,16 +177,19 @@ void shouldFailToLoadEntitlementContextIfClaimNotFound() {
         lenient().when(paymentCycleRepository.findById(currentPaymentCycleId)).thenReturn(Optional.of(currentPaymentCycle));
         lenient().when(paymentCycleRepository.findById(previousPaymentCycleId)).thenReturn(Optional.of(previousPaymentCycle));
         DetermineEntitlementMessagePayload payload = buildEntitlementPayload(claimId, previousPaymentCycleId, currentPaymentCycleId);
+        given(payloadMapper.getPayload(any(), eq(DetermineEntitlementMessagePayload.class))).willReturn(payload);
```

Comment:
can this use any() instead of eq()?

Reply:
Sadly we have to use `eq` here because that parameter is a `Class`.